Assets/Scripts/MouseTracker.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseTracker : MonoBehaviour {
```

Comment:
What's this class for? Add a summary (type three `/`'s in visual studio for the shortcut. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/summary